openvpn-install.sh

```diff
@@ -133,6 +133,7 @@ if [[ -e /etc/openvpn/server.conf ]]; then
 			rm -rf pki/issued/$CLIENT.crt
 			rm -rf /etc/openvpn/crl.pem
 			cp /etc/openvpn/easy-rsa/pki/crl.pem /etc/openvpn/crl.pem
+			chmod nobody:nobody /etc/openvpn/crl.pem
```

Comment:
You should change your patch or this owner of this repo can.
I'm not the owner so I can't push code on this repo.

So the best way is you change your patch :)

Thank you